tests/integration/test_crossmodel.py

```diff
@@ -96,9 +96,9 @@ async def test_remove_saas(event_loop):
 async def test_add_relation_with_offer(event_loop):
     async with base.CleanModel() as model_1:
         application = await model_1.deploy(
-            'cs:mysql-58',
+            'ch:mysql',
             application_name='mysql',
-            series='bionic',
+            series='xenial',
```

Comment:
I'd rather we didn't go backward here. Can we pick a better charm or better series?

Reply:
Oh looks like I mixed up the version names again, for some reason that's a problem for me :) gotta keep a post-it or something